WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -6,16 +6,45 @@ class GutenbergViewController: UIViewController, PostEditor {
 
     let errorDomain: String = "GutenbergViewController.errorDomain"
 
-    private struct Analytics {
-        static let editorSource = "gutenberg"
-    }
-
     enum RequestHTMLReason {
         case publish
         case close
         case more
     }
 
+    // MARK: - UI
+
+    private lazy var titleStackView: UIStackView = {
+        let leftView = UIView()
+        leftView.translatesAutoresizingMaskIntoConstraints = false
+        leftView.heightAnchor.constraint(equalToConstant: Size.titleTextFieldHeight).isActive = true
+        leftView.widthAnchor.constraint(equalToConstant: Size.titleTextFieldLeftPadding).isActive = true
+        leftView.backgroundColor = Colors.background
+        let stackView = UIStackView(arrangedSubviews: [leftView, titleTextField])
+        stackView.axis = .horizontal
+        return stackView
+    }()
+
+    private lazy var titleTextField: UITextField = {
+        let textField = UITextField()
+        textField.borderStyle = .none
+        textField.heightAnchor.constraint(equalToConstant: Size.titleTextFieldHeight).isActive = true
+        textField.font = Fonts.title
+        textField.textColor = Colors.title
+        textField.backgroundColor = Colors.background
+        return textField
+    }()
+
+    private lazy var separatorView: UIView = {
+        let view = UIView()
+        view.heightAnchor.constraint(equalToConstant: Size.titleTextFieldBottomSeparatorHeight).isActive = true
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = Colors.separator
+        return view
+    }()
+
+    // MARK: - PostEditor
```

Comment:
YW! 

PostEditor has lots of "var"s so even if we move PostEditor conformation to an extension we'll need lots of stored variables in class itself(since we can't define stored variables in extensions), so I am not sure if that'll really clean things, it might also be duplicating lots of stuff, but it is worth trying and seeing in the future. 👍